index.html

```diff
@@ -181,7 +185,9 @@ <h4 class="work-title">Multi Post Stories</h4>
               <li class="tech-used">Bootstrap</li>
               <li class="tech-used">Ruby</li>
             </ul>
-            <button class="see-project-btn">See Project</button>
+            <button id="see-project-btn" class="see-project">
+              See Project
+            </button>
```

Comment:
> You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.

- [ ] According to the project requirement, you are meant to remove the hard-coded projects in your index.html file and generate it randomly using the array of projects in your `script.js` file. I suggest you implement this in your project as its part of this milestone. :+1: :100: